.circleci/config.yml

```
@@ -33,30 +33,63 @@ commands:
-            cleanup() { docker rm -f << parameters.name >> >/dev/null 2>&1 || true; }
+            cleanup() { docker rm -f << parameters.container_name >> >/dev/null 2>&1 || true; }
             trap cleanup EXIT
-            docker run -d --rm -p << parameters.host_port >>:<< parameters.port >> --name << parameters.name >> << parameters.image >>
+            # Check if the image exists before running container
+            echo "🔍 Checking if image << parameters.image >> exists..."
+            if ! docker image inspect << parameters.image >> >/dev/null 2>&1; then
+              echo "❌ Image << parameters.image >> not found! Available images:"
+              docker images
+              exit 1
+            fi
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)